features/fixtures/maze_runner/Assets/Scripts/Main.cs

```diff
@@ -67,6 +81,59 @@ public void Start()
         InvokeRepeating("DoRunNextMazeCommand",0,1);
     }
 
+    // example command: RunOnTarget.exe 0x01004B9000490000 --no-wait -- --mazeIp 192.168.0.whatever --cacheType i --cacheIndex 3 --mountName BugsnagCache
+    private void GetSwitchArguments()
+    {
+#if UNITY_SWITCH
+        int count = bugsnag_getArgsCount();
+        Debug.Log("args count: " + count);
+
+        for (int i = 0; i < count; i++)
+        {
+            var arg = bugsnag_getArg(i);
+            if (!arg.Contains("--"))
+            {
+                continue;
+            }
+            Debug.Log("CHECKING ARG: " + arg);
+            if (arg == "--mazeIp")
+            {
+                var ip = bugsnag_getArg(i + 1);
+                _mazeHost = "http://" + ip + ":9339";
+                Debug.Log("SET MAZE HOST TO: " + _mazeHost);
+            }
+            if (arg == "--cacheType")
+            {
+                var cacheType = bugsnag_getArg(i + 1);
+                switch (cacheType)
+                {
+                    case "r":
+                        _switchCacheType = SwitchCacheType.R;
+                        break;
+                    case "i":
+                        _switchCacheType = SwitchCacheType.I;
+                        break;
+                    case "n":
+                        _switchCacheType = SwitchCacheType.None;
+                        break;
+                }
```

Comment:
I wouldn't want to go overboard on input validation when we're in control of it, but a `default:` here that says the argument is invalid and bails out could avoid some unexpected behaviour and lost time.

Reply:
Added